tests/acceptance.suite.yml

```diff
@@ -0,0 +1,13 @@
+# Codeception Test Suite Configuration
+#
+# Suite for acceptance tests.
+# Perform tests in browser using the WPWebDriver or WPBrowser.
+# Use WPDb to set up your initial database fixture.
+# If you need both WPWebDriver and WPBrowser tests - create a separate suite.
+
+actor: AcceptanceTester
+modules:
+    enabled:
+        - WPDb
+        - WPBrowser
```

Comment:
I think for acceptance tests this should be `WPWebDriver`, functional tests can use `WPBrowser`.